### DISCUSSION TITLE
可以抓取 pdf 文件吗？要怎么设置？

### DISCUSSION THREAD

**Discussion by tjulyang** [xifangczy/cat-catch#165] [Q&A]
试了在抓取类型添加 application/pdf ，刷新网页后仍然抓不到 PDF 文件

**Comment by xifangczy** [comment]
猫抓原理不是分析网页内容 是监听浏览器请求 如果你没打开pdf的动作 抓不到的。

**Reply by tjulyang** [reply]
本来想实现功能是能提取 pdf 文件，这样就不用使用开发者模式去找 pdf 了。我纯小白，不太懂这些，非常感谢你的回答。
